src/CmsmsMessage.php

```diff
@@ -53,6 +63,11 @@ public function reference(string $reference): self
         return $this;
     }
 
+    public function getReference(): string
+    {
+        return $this->reference;
+    }
+
     public function tariff(int $tariff): self
```

Comment:
Think we can remove all tariff stuff as it doesn't seem supported anymore (see other comment as well)